profile/views.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth import get_user_model
+from rest_framework import viewsets
+from .serializers import UserSerializer
+User = get_user_model()
+
+
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
Does this API need any kind of authentication? What about authorization, is anyone able to create/modify/delete any users?

Reply:
Good point, there is no authentication verification. This API was just a CRUD, it was because of that dj-rest-auth was installed. This API will be deprecated soon.